src/api/models/themes.js

```diff
@@ -9,29 +9,86 @@ import deepClone from 'lodash/cloneDeep';
 import get from 'lodash/get';
 import ejs from 'ejs';
 
-const renderFile = promisify(ejs.renderFile);
+// --- Global variable for the Theme system
+export const THEMES_VERSION = '3';
+export const THEMES_FOLDER = '../../app/custom/themes';
 
+// --- Global function for the Theme system
+const renderFile = promisify(ejs.renderFile);
 const logger = getLogger('system');
 
+// --- TypeScript without TypeScript (Required an ide with TypeScript support)
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏I like that!

Reply:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** Add the `checkJs` setting to `.vscode/settings.json`

See https://github.com/Inist-CNRS/ezs/blob/master/.vscode/settings.json#L2‏